functions-and-operators/string-functions.md

```diff
@@ -107,19 +107,134 @@ Return a string containing binary representation of a number.
 
 ### [`BIT_LENGTH()`](https://dev.mysql.com/doc/refman/8.0/en/string-functions.html#function_bit-length)
 
-Return length of argument in bits.
+The `BIT_LENGTH()` function is used to return the length of a given argument in bits. It calculates the length of the argument and then returns the result in bits.
+
+Examples:
+
+```sql
+SELECT BIT_LENGTH("TiDB");
++----------+
+| BIT_LENGTH("TiDB") |
++----------+
+|  32 |
++----------+
+```
+
+8 bits per character x 4 characters = 32 bits
+
+```sql
+SELECT CustomerName, BIT_LENGTH(CustomerName) AS BitLengthOfName FROM Customers;
+| CustomerName|BitLengthOfName |
+|-------------|----------------|
+| Albert Einstein | 120 |
+| Robert Oppenheimer | 144 |
+```
+
+```sql
+SELECT BIT_LENGTH("PingCap 123");
++----------+
+| BIT_LENGTH("PingCap 123") |
++----------+
+| 88 |
++----------+
+```
+
+8 bits per character (space is counted as it is a non-alphanumeric character) x 11 characters = 88 bits
+
+> **Note:**
+>
+> The second example operates under the assumption that there is a database with a row titled `Customers` and a column inside titled `CustomerName`
```

Comment:
Still sounds strange here, since column inside row inside table inside database, "database with a row" is strange here.

Reply:
I'm not sure if I explained it cear here. Database is a top entity, which Table is a inner entity inside Database. And Row is a inner entity inside Table. Thus there is Table between Database and Row, and Row doesn't have names while Table and Columns do.

Reply:
got your point, now fixed. you can re-read it